### PR TITLE
Use descriptive names for constants

### PR DESCRIPTION
**Pull request by pokrakam**
There is still a lot of code being written in the style of `CONSTANTS c_01 ... VALUE '01'.`,  I suggest this should be emphasised in its own section. 
We could put this in the "Use descriptive names'' section too, but I feel it is better among constants because it also hilights that we should look to enumeration classes first.

### CODE REVIEW

**Review by schmelto — APPROVED** [review]

**Comment by HrFlorianHoffmann** [comment]
Yes, I agree. This point could need some repetition. Two suggestions:
- Let's make this the second sub-section inside the _Constants_ section, directly after _Use constants instead of magic numbers_ It fits better at the front there because it has more a general or introductive character and should come before we dive into the technical details of how to implement in the other two sub-sections. It also fits in nicely beside the general observation to not use magic numbers. 
- Let's add a link to the section _Use descriptive names_. Maybe a quoted paragrapth at the end:
> This is a repetition of the rule [Use descriptive names](#use-descriptive-names), specifically for constants.
@pokrakam Can you do these changes in your original branch? It's not inside this repository, so I cannot directly edit it. Or should I move the suggested changes over and add them from an own branch?

**Review by HrFlorianHoffmann — CHANGES_REQUESTED** [review]
⬆

**Comment by pokrakam** [comment]
@HrFlorianHoffmann sure I can update this, bit busy  but will get to it in the next couple of days.

File: clean-abap/CleanABAP.md [hunk 1]
```diff
@@ -866,6 +867,26 @@ ENDDO.
...
+CONSTANTS: 
+  c_01 TYPE spart VALUE '01',
+  c_mmsta TYPE mmsta VALUE '90'.
+```
+There is little benefit to either variant. It is not informative for the reader, and if 
+the value ever needs to change then a constant that parrots its value must also be renamed. 
+
+If a constant is declared in code then it should describe its meaning not its content.
+```ABAP
+CONSTANTS c_status_inactive TYPE mmsta VALUE '90'.
```
Comment (ConjuringCoffee) [inline clean-abap/CleanABAP.md:887]: The "good" example should not contain any prefix, see rule "Avoid encodings, esp. Hungarian notation and prefixes"

**Pull request closed without merging**